### PR TITLE
Update GitHub app docs

### PR DESCRIPTION
**Pull request by djmitche**
/cc @edunham

### CODE REVIEW

File: ui/docs/manual/deploying/README.mdx [hunk 1]
```diff
@@ -46,6 +46,15 @@ Taskcluster can integrate with GitHub in two complementary ways:
 * The taskcluster-github service provides a GitHub App which can create tasks in response to events in GitHub repositories. See [Github Integration](/docs/manual/deploying/github) for more details.
 * The web-server service supports user logins via GitHub.  See [Login Strategies](/docs/manual/deploying/login-strategies) for more details.
 
+### Why Are There Two Aapps?
```
Comment (imbstack) [inline ui/docs/manual/deploying/README.mdx]: Aaaaappppssss

Comment (helfi92) [reply ui/docs/manual/deploying/README.mdx]: Haha!

File: ui/docs/manual/deploying/login-strategies.mdx [hunk 2]
```diff
@@ -30,6 +30,7 @@ JSON *requires* the use of the double-quote character (`"`), but shells, YAML pa
 
 In order to enable the GitHub login strategy, specify the GitHub client ID and secret for an OAuth application created for use against this service and its web UI.
 Create it under "Settings" (for yourself or for an org) -> "Developer Settings" -> "OAuth Apps" -> "New OAuth App".
+The name for the app is not critical, but we suggest something like "deploymentname-taskcluster-signin".
```
Comment (helfi92) [inline ui/docs/manual/deploying/login-strategies.mdx]: Should this be `"<deployment-name>-taskcluster-signin"`?

Comment (djmitche) [reply ui/docs/manual/deploying/login-strategies.mdx]: Thanks :D

**Pull request merged**